migration-engine/core/src/commands/schema_push.rs

```diff
@@ -41,6 +41,7 @@ impl MigrationCommand for SchemaPushCommand {
                 tracing::warn!(unexecutable = ?checks.unexecutable_migrations, "Aborting migration because at least one unexecutable step was detected.")
             }
             (0, 0, _) | (0, _, true) => {
+                println!("{:?}", applier.render_steps_pretty(&database_migration));
```

Comment:
this is going to interfere with JSON-RPC

Reply:
Jep, removed it.